spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -467,76 +471,111 @@ public void testDataFilesNotRewrittenWithLowDeleteRatio() throws Exception {
   }
 
   @TestTemplate
-  public void testBinPackWithDeletes() throws IOException {
-    assumeThat(formatVersion).isGreaterThanOrEqualTo(2);
+  public void testBinPackWithDVs() throws IOException {
```

Comment:
the diff seems messed up. 

I assume you just renamed this test to `testBinPackWithV2PositionDeletes` test below and added this new `testBinPackWithDVs`.

Ideally, this line should only diff on the renaming. then a new test method `testBinPackWithDVs` was added after this method.

Reply:
Sure let me reorder this so the V2 test comes first, I think that should make the diff a bit less complicated to look at

Reply:
I reordered it, the if format >= 3 /else from the previous diff will still show since the previous test tested both but I do think the diff is a bit easier to look at now.